scripts/test_checkpoint_integration.py

```
@@ -0,0 +1,210 @@
+#!/usr/bin/env python3
```

Ruff (0.14.6)

Shebang is present but file is not executable

(EXE001)